tests/src/ExistingSite/InstallStateTest.php

```diff
@@ -241,37 +240,41 @@ private function createMedia(array $values = []) {
    * Check if newly created content type's content can be
    * cloned by user or not.
    */
+  // @todo The action buttons disappeared on the node edit page, isue raised
+  // Issue - https://www.drupal.org/project/gin/issues/3410516
+  /*
   public function testEntityCloneForNewContentType(): void {
-    // Create new content type.
-    $test_node_type = NodeType::create([
-      'type' => 'test_node',
-      'name' => 'Test node type',
-    ]);
-    $test_node_type->save();
-    // Mark test entity for clean up at the end.
-    $this->markEntityForCleanup($test_node_type);
-
-    $assert_session = $this->assertSession();
-    $account = $this->createUser();
-    $account->addRole('content_administrator');
-    $account->save();
-    $this->drupalLogin($account);
-
-    // Create a node of test_node type.
-    $node_page = $this->createNode([
-      'type' => 'test_node',
-      'title' => 'Categories Page',
-      'uid' => $account->id(),
-      'moderation_state' => 'published',
-    ]);
-
-    // Visit node edit page created above.
-    $this->drupalGet($node_page->toUrl('edit-form'));
-    $assert_session->statusCodeEquals(200);
-
-    // Assert clone tab exists.
-    $assert_session->linkExists('Clone');
+  // Create new content type.
+  $test_node_type = NodeType::create([
+  'type' => 'test_node',
```

Comment:
PHPCS issues

Reply:
There isn't any PHPCS warning/error in CI or on Local.